extensions/adobe/experience/adcloud/syncedremarketingaudiences.example.1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "xdm:segmentId": "https://data.adobe.io/entities/aam-segment/4238827",
+  "xdm:segmentType": 3,
+  "xdm:timestamp": "2018-08-07T08:06:34.672Z",
```

Comment:
This timestamp might be better in partner details to signify when the sync happened with the partner since it could have happened for one and not the other